src/pages/learn/index.tsx

```diff
@@ -0,0 +1,943 @@
+                    The Cryptopians: Idealism, Greed, Lies, and the Making of
+                    the First Big Cryptocurrency Craze
+                  </Link>{" "}
+                  <em>February 22, 2022 - Laura Shin</em>
+                </li>
+                <li>
+                  <Link to="https://www.goodreads.com/book/show/55360267-out-of-the-ether">
+                    Out of the Ether: The Amazing Story of Ethereum and the $55
+                    Million Heist that Almost Destroyed It All
+                  </Link>{" "}
+                  <em>September 29, 2020 - Matthew Leising</em>
+                </li>
+                <li>
+                  <Link to="https://www.goodreads.com/en/book/show/50175330-the-infinite-machine">
+                    The Infinite Machine: How an Army of Crypto-hackers Is
+                    Building the Next Internet with Ethereum
+                  </Link>{" "}
+                  <em>July 14, 2020 - Camila Russo</em>
+                </li>
+                <li>
+                  <Link to="https://www.goodreads.com/en/book/show/22174460-the-age-of-cryptocurrency">
+                    The Age of Cryptocurrency: How Bitcoin and the Blockchain
+                    Are Challenging the Global Economic Order
+                  </Link>{" "}
+                  <em>January 12, 2016 - Paul Vigna, Michael J. Casey</em>
+                </li>
+                <li>
+                  <Link to="https://www.goodreads.com/en/book/show/34964890-the-truth-machine">
+                    The Truth Machine: The Blockchain and the Future of
+                    Everything
+                  </Link>{" "}
+                  <em>February 27, 2018 - Paul Vigna, Michael J. Casey</em>
+                </li>
+                <li>
+                  <Link to="https://www.goodreads.com/book/show/23546676-digital-gold">
+                    Digital Gold: Bitcoin and the Inside Story of the Misfits
+                    and Millionaires Trying to Reinvent Money
+                  </Link>{" "}
+                  <em>May 24, 2021 - Nathaniel Popper</em>
+                </li>
+                <li>
+                  <Link to="https://www.goodreads.com/en/book/show/56274031-kings-of-crypto">
+                    Kings of Crypto: One Startup&#39;s Quest to Take
+                    Cryptocurrency Out of Silicon Valley and Onto Wall Street
+                  </Link>{" "}
+                  <em>December 15, 2020 - Jeff John Roberts</em>
+                </li>
+                <li>
+                  <Link to="https://github.com/ethereumbook/ethereumbook">
+                    Mastering Ethereum
+                  </Link>{" "}
+                  <em>
+                    December 23, 2018 – Andreas M. Antonopoulos, Gavin Wood
+                    Ph.D.{" "}
+                  </em>
+                </li>
+              </ul>
+              <p>Podcasts covering Ethereum and Cryptocurrencies:</p>
```

Comment:
This should be a heading

```suggestion
              <h3>Podcasts covering Ethereum and Cryptocurrencies:</h3>
```